crates/lsp/src/lib.rs

```diff
@@ -520,7 +520,7 @@ fn on_did_save_text_document(
     }
 
     // We need to refresh lenses when we compile since that's the only time they can be accurately reflected
-    let _ = state.client.code_lens_refresh(());
+    std::mem::drop(state.client.code_lens_refresh(()));
```

Comment:
What was the rational clippy gave for this change?

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#/let_underscore_future

Reply:
Ahh its because its a Future -- was just skimming and thought it would be weird if it was for a non-async function.